src/components/booking-details/TripDetailsCard.tsx

```diff
@@ -0,0 +1,40 @@
+import { TripDetailsCardProps } from '@/types/booking';
+import { ArrowRightOutlined } from '@ant-design/icons';
+import { Card } from 'antd';
+import dayjs from 'dayjs';
+import './TripDetailsCard.less';
+
+const TripDetailsCard = ({ booking }: TripDetailsCardProps) => {
+  return (
+    <div>
+      <Card title="Trip Details">
+        <p className="carrier-name">carrier name : {booking.carrier_name}</p>
+        <div className="divider"></div>
```

Comment:
nit: You may also use Divider component from Ant

Reply:
Thanks for your suggestion. but I just want to customize the divider, that's why I use this one!